src/pretix/base/settings.py

```diff
@@ -2001,6 +2008,24 @@ def unserialize(cls, s):
             '_scheme': 'full_transcription',
         },
     }),
+    ('title_salutation_given_family', {
+        'fields': (
+            ('salutation', pgettext_lazy('person_name', 'Salutation'), 1),
+            ('title', pgettext_lazy('person_name', 'Title'), 1),
+            ('given_name', _('Given name'), 2),
+            ('family_name', _('Family name'), 2),
+        ),
+        'concatenation': lambda d: ' '.join(
+            str(p) for p in (d.get(key, '') for key in ["salutation", "title", "given_name", "family_name"]) if p
```

Comment:
Just thought about not having the salutation here, as it's probably only used in Mail templates? @raphaelm

Reply:
Thought about this as well and I think it's a good idea.

Reply:
:+1: